app/src/main/java/com/cornellappdev/uplift/ui/components/Home/HomeCard.kt

```diff
@@ -0,0 +1,128 @@
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.ui.Alignment.Companion.TopEnd
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import com.cornellappdev.uplift.R
+import com.cornellappdev.uplift.models.Gym
+import com.cornellappdev.uplift.util.montserratFamily
+
+
+@Composable
+    fun homeCard(gymexample:Gym ) {
+    var paintres = remember  {mutableStateOf(if(gymexample.isFavorite()) R.drawable.ic_star_filled else R.drawable.ic_star) }
+        Box(modifier= Modifier
+            .padding(5.dp)
+            .fillMaxWidth()
+            .fillMaxHeight()
+            .aspectRatio(2F / 1F)){
+            Card(
+                modifier = Modifier
+            ) {
+                Column() {
+                    Column(modifier = Modifier.weight(3F)) {
+                        Image(
+                            modifier = Modifier.fillMaxWidth(),
+                            painter = painterResource(id=R.drawable.court_lines),
+                            contentDescription = "Gym Image",
+                            contentScale = ContentScale.FillWidth
+                        )
+
+                    }
+                    Column(modifier = Modifier.weight(2F)) {
+                        Row(
+                            modifier = Modifier.padding(5.dp)
+                        ) {
+                            Text(
+                                text =gymexample.name,
+                                fontSize = 20.sp,
+                                fontFamily = montserratFamily
+
+                                )
+                            Spacer(modifier = Modifier.weight(1F))
+                            Image(
+                                painter = painterResource(id = R.drawable.ic_dumbbell),
+                                contentDescription = "Dumbell",
+                            )
+                            Image(
+                                painter = painterResource(id = R.drawable.ic_bowling_pins),
+                                contentDescription = "Bowling Pins",
+                            )
+
+                        }
+                        Row(modifier = Modifier.padding(start = 5.dp, end = 5.dp)) {
+                            Text(
+                                text = "Open",
+                                fontSize = 12.sp,
+                                color = Color.Green,
+                                fontFamily = montserratFamily
+
+                                )
+                            Spacer(modifier = Modifier.width(10.dp))
+                            Text(
+                                text = "Closes at 6 pm",
+                                fontSize = 12.sp,
+                                fontFamily = montserratFamily
+                                )
```

Comment:
I believe I have provided the sufficient functionality for you to make these dynamically update correctly to a `Gym`'s data. See my function `isCurrentlyOpen()` in `Functions.kt` for determining open/closed status, and change this to update to if a Gym is open or closed at the current time (with correct colors, too). To get a gym's closing time, you can get the last `TimeInterval` appearing in today's `TimeInterval` list. You can get the index of today by doing:

`val day = ((Calendar.getInstance().get(Calendar.DAY_OF_WEEK) - 2) + 7) % 7`

(I should have made a helper function for getting this index, now that I think of it...)